### ISSUE TITLE
IndexError in KneeLocator after upgrade to 0.5.3

### ISSUE THREAD

**Issue by tommilligan** [arvkevi/kneed#64]
After upgrading to 0.5.3 from 0.5.1, I get an `IndexError` on an existing case. I didn't expect there to be a change in behaviour, as it was only a patch version bump.

```log
>       self.knee_y = self.y[self.x == self.knee][0]
E       IndexError: index 0 is out of bounds for axis 0 with size 0
```

The line throwing the error is [here](https://github.com/arvkevi/kneed/compare/v0.5.1...v0.5.3#diff-3087e28ef7b2cb52d4fcfec312fbf6d0R96)

**Comment by tommilligan** [comment]
There is a reproducible test case in #65

**Comment by arvkevi** [comment]
Thanks, @tommilligan, good catch! I added functionality to output the `y` value at the knee point. In this example, there is no knee, which is throwing the `IndexError`.

Something like this would be great [here](https://github.com/arvkevi/kneed/blob/76652ce576b75e97dcc3377d07f8d88b954e368d/kneed/knee_locator.py#L95-L97):

```python
# Step 6: find knee
self.knee_y, self.norm_knee_y = None, None
self.knee, self.norm_knee = self.find_knee()
if self.knee:
    self.knee_y = self.y[self.x == self.knee][0]
    self.norm_knee_y = self.y_normalized[self.x_normalized == self.norm_knee][0]
```

If you want to add it on #65 and update the test to assert that `self.knee` and `self.knee_y` are both `None`.

Probably should've bumped the version after adding the `y` functionality, so we can do it after this PR.

**Comment by tommilligan** [comment]
Thanks, that makes sense - I've updated the PR as you described. Thanks for keeping this up to date!

**Comment by arvkevi** [comment]
fixed in #65
